src/models/room-state.ts

```diff
@@ -53,13 +53,15 @@ enum OobStatus {
 export interface IPowerLevelsContent {
     users?: Record<string, number>;
     events?: Record<string, number>;
+    notifications?: Record<'room', number>;
```

Comment:
This type isn't right strictly speaking, as it would render a value of `{}` invalid.

```suggestion
    notifications?: {
        room?: number;
    };
```